xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+</para>
+
+  <procedure>
+   <step>
+    <para>
+      Under <literal>Device Type</literal> select VLAN.
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Set the <literal>Configuration Name</literal> to match what has been defined in the ACI fabric. In the example network that is <literal>vlan.4093</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+        Press F10 (Next) to return to the <literal>Network Card Setup | Address screen.</literal>
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+       Select the Ethernet device that is connected to the ACI fabric  for <literal>Real Interface for VLAN</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+       Enter your VLAN ID, which in the example network is 4093.
+    </para>    
+   </step> 
+    
+   <step>
+    <para>
+      Enable <literal>Dynamic Address - DHCP</literal>.
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+      Change to the General tab and set the MTU to 1600.
+    </para>
+   </step> 
+
+    <step>
+    <para>
+      Press F10 to save changes for new VLAN device, and press F10 again to exit Network Settings.
+    </para>
+   </step>    
+</procedure>
+
+
+<para>
+   Now that you have made these changes, re-enter Network Settings and configure routing. 
+</para>
+
+<procedure>
+   <step>
+    <para>
+     Go back into Network Settings (Click Continue when you see the Chef client warning).
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Change to the Routing tab and add these entries to the Routing Table:
+        
+        <command>Destination     224.0.0.0
+        Genmask         240.0.0.0
+        Gateway         0.0.0.0
+        Device          vlan.4093 </command>(Select the VLAN device created in the previous steps)
+    </para>
+   </step> 
+   
+    <step>
+    <para>
+     Press F10 to save and exit Network Settings, then press F9 to exit Yast.
+    </para>
+   </step> 
+ </procedure>
+</sect1>
+
+<sect1 xml:id="sec.connectivity">
+  <title>Confirm Correct Connectivity</title>
+
+  <para>
+      Edit the <filename>/etc/dhclient.conf</filename> file on all compute nodes to verify that <literal>send dhcp-client-identifier</literal> has the MAC address of the NIC on the ACI fabric. (Run <command>ip addr show</command> to find the MAC address.) It should look like this, uncommented, with <literal>1:</literal> prepended to your MAC address, followed by a semi-colon, like this example: 
+   </para>
+   
+    <para> 
+        <literal>  
+     send dhcp-client-identifier 1:0:a0:24:ab:fb:9c;
+     </literal>
+  </para>
+  
+  <para>
+     Now install and run <command>lldpd</command>, the Link Layer Discovery Protocol daemon. You will use this to verify correct connectivity between your compute nodes.
+ </para>
+ 
+ <screen><prompt>root #  </prompt><command>zypper in lldpd</command> 
```

Comment:
... _not_ followed by  a space. Also there si no need to use command in screens:

    <screen>&prompt.root;zypper in lldpd</screen>

However, we are supposed to use sudo whenever possible (rather than requiring to log in as root:

       <screen>&prompt.user;sudo zypper in lldpd</screen>

_However_ using a prompt is usually only required if we want to display the command and its output, to make it possible to distinguis between the command and the output. To cut a long story short, the correct way to document these three commands is simply:

   <screen>sudo zypper in lldpd
    sudo systemctl start lldpd.service
    sudo lldpctl</screen>